src/views/canvas/CanvasSeqBlock.js

```diff
@@ -377,7 +395,7 @@ const View = boneView.extend({
     return this.throttledDraw();
   },
 
-  _getClickPos: function(e) {
+  _getResidueAtMouseEvent: function(e) {
```

Comment:
renamed `_getClickPos` to `_getResidueAtMouseEvent` since
1. The helper figures out the residue at current mouse location
2. Works for any mouse event including clicks and mouse movement